cpp/include/cudf/io/json.hpp

```diff
@@ -130,6 +130,18 @@ class json_reader_options {
   // Whether to recover after an invalid JSON line
   json_recovery_mode_t _recovery_mode = json_recovery_mode_t::FAIL;
 
+  // Validation checks for spark
+  // Should the json validation be strict of not
```

Comment:
I agree with the other reviewers. A blurb clarifying definition of strictness (e.g. compliance with the JSON spec, null interpretations, etc.)
```suggestion
  // Should the json validation be strict or not
```